### ISSUE TITLE
(N64 Sound Tool) Acclaim "DEAN" audio doesn't decompress properly

### ISSUE THREAD

**Issue by turboboy215** [jombo23/N64-Tools#57]
When I try to extract the compressed samples from Shadow Man or South Park, they all sound very sped up, despite having a fixed sample rate in the N64 Sound Tool. Putting the sample rate in half or lower makes the samples seem at the right speed, but they sound very bad quality, much worse than in-game. For example, the South Park intro music uses a rate of 11khz, but it only sounds right if you change it to 5.5khz. Shadow Man's voices sound even more sped up, possibly at 4x speed. This may be due to an issue of sounds being exported at the incorrect bitrate or an endianness issue.

Also, the music samples of Earthworm Jim 3D are not actually raw PCM, but they are a type of compressed sample format that resembles 4-bit PCM, but is a different lossy encoding, so this should be acknowledged at some point in the future.

**Comment by SubDrag** [comment]
Yeah games will have individual sound frequencies. It was not feasible to go through every game and figure out their scheme. You'll have to sort out individually.

Also are you sure about Earthworm Gym? It sounded right and I reverse engineered it to the audio I though and it was just fed directly to rsp that way.

**Comment by turboboy215** [comment]
I'm pretty sure the Acclaim sounds are still incorrectly ripped, but use the correct sample rate, because in-game it sounds much clearer than when it is outputted through the program. The SP intro definitely sounds 11khz to me. Like I said, the game might be using a lower bitrate than extracted or something like that.

I'm not entirely sure about EWJ3D, but to me, it doesn't seem quite right. The music in that game also sounds higher quality than 11khz, whereas in the program, it sounds too fast.

**Comment by SubDrag** [comment]
Acclaim seemed ok just wrong sample rate, so not sure where they pull it from then; clearly not header.

You're right EWJ3D doesn't sound perfect.  Not sure why though, I had checked and fed straight to RSP though, which is odd.
